lib/tests/basic-call.rb

```diff
@@ -87,6 +92,42 @@
 
 end
 
+TestDefinition.new("Basic Call - Tel URIs") do |t|
+  caller = t.add_endpoint
+  callee = t.add_endpoint
+
+  ringing_barrier = Barrier.new(2)
```

Comment:
This test could break confusingly if it's run on a system with ENUM enabled. Can you add a comment (just in the code) suggesting that the user check their enum rules first if this test fails